quick-start-guide/src/main/resources/QSG/bin/qsg.bat

```diff
@@ -480,7 +484,7 @@ set request_data=%~3\add-role.xml
 echo(
 echo Creating a user named cameron...
 
-REM The following command can be used to create a user.
+REM The following command can be used to create a user cameron.
```

Comment:
change to Cameron (with uppercase C). Change all applicable places

Reply:
The username of the user is cameron, Hence used "cameron"